package.json

```diff
@@ -31,7 +32,10 @@
     "@babel/preset-react": "7.10.4",
     "@babel/runtime": "7.11.2",
     "@testing-library/react-hooks": "3.4.2",
-    "jest": "26.4.2",
+    "@types/react": "^17.0.0",
```

Comment:
Can we pin these dependencies to the exact version by reinstalling with `-E`. So this becomes `17.0.0`, as well as the other dependencies added below. This will satisfy Renovate 👍🏻

Reply:
Ran:
`npm un @babel/preset-typescript rollup-plugin-typescript2 @types/typescript @types/react-dom @types/react jest typescript`

`npm i -E -D @babel/preset-typescript rollup-plugin-typescript2 @types/typescript @types/react-dom @types/react jest typescript`